### PR TITLE
fix: channel id parsing

### PR DESCRIPTION
**Pull request by dickie81**
## Description:
Actual end to end working on device.

### Checklist for this pull request

* [X] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation included
* [X] One of:
  * [X] The PR is not making any UI change
  * [ ] The PR is making a UI change but not a product change
    * [X] Screenshots included

Thank you!

### CODE REVIEW

**Comment by dickie81** [comment]
screen shot 2018-06-11 at 14 41 36

File: src/player/APVideoPlayer.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const APReactVideoView = {
   name: 'ReactVideoView',
   propTypes: {
     src: PropTypes.shape({
-      id: PropTypes.string,
+      id: PropTypes.number,
```
Comment (Budaa) [inline src/player/APVideoPlayer.js:28]: Wouldn't it be better to use strings instead? As this is a reusable component, other players may use ids that aren't numbers.

Comment (dickie81) [reply src/player/APVideoPlayer.js:28]: Native complains that it wants a double

Comment (Budaa) [reply src/player/APVideoPlayer.js:28]: Alright, so it looks like aligning the current implementation. Shouldn't be a problem to refactor when necessary. LGTM

**Review by Budaa — APPROVED** [review]
Looking good.

**Pull request merged**